frontend/javascripts/messages.js

```diff
@@ -266,13 +266,15 @@ instead. Only enable this option if you understand its effect. All layers will n
   "users.needs_admin_rights": "Admin rights are required to change the permissions of users.",
   "users.multiple_selected_users":
     "You selected more than one user. To change the permissions of users you need to select them individually.",
-  "users.grant_admin_rights_title": "Do you really want to grant admin rights?",
-  "users.grant_admin_rights": _.template(
-    "You are about to grant admin privileges to <%- numUsers %> user(s) giving them access to all teams, datasets and annotations. Do you want to proceed?",
+  "users.change_permissions_title": "Do you really want to change the permissions of this user?",
+  "users.revoke_all_permissions": _.template(
+    "Ths user <%- userName %> will no longer have any extra access permissions on datasets.",
```

Comment:
Why the "on datasets"? This message will also be shown when changing from Admin status to Member and Admins have many more permissions, so I would probably change it to "will no longer have any extra permissions."